ethstorage/es-node
# Enable Metrics in the es-node

Add metrics params to the command to enable metrics and show them to a report.

How to use it:
Add --metrics.addr --metrics.port --metrics.enabled to enable and set the metrics report address or port.
`--network devnet --datadir ./database --storage.files storage.dat --l1.beacon http://65.108.236.27:5052  --l1.rpc http://65.108.236.27:8545 --storage.l1contract 0x882BC290fc22C330592819977c48968a62AE25f4 --storage.miner 0x5C935469C5592Aeeac3372e922d9bCEabDF8830d --l1.beacon-based-time 1695981612 --l1.beacon-based-slot 1 --p2p.listen.udp 30305 --log.level debug --metrics.addr <your address> --metrics.enabled --p2p.bootnodes enr:-Li4QPWz6DaLXwZ3vgUfx5i0M6ysMAhjlFrkLlzaKdvbMykrIzdncYXQG0rjF8anZfv65LpEuYVdOZPbjzwvthqUIq-GAYsoHnk0imV0aHN0b3JhZ2XbAYDY15SIK8KQ_CLDMFkoGZd8SJaKYq4l9MGAgmlkgnY0gmlwhEFs7BuJc2VjcDI1NmsxoQI2rSdv0CGNPia7uQgvYdt4XuW2Nkp062a0jsr-H-SJ4IN0Y3CCJAaDdWRwgnZh`

How to verify it:
After the node starts, see the report in `http://<your address>:7300`
see `es_node_default_+` data from the page.

[sample report](http://65.109.63.154/d/fdfcc57c-30fb-424e-8dc5-b0cf7bcce84b/ethstorage-monitoring?orgId=2)
